server/app/src/androidTest/java/sh/calaba/instrumentationbackend/StatusReporter.java

```diff
@@ -1,26 +1,20 @@
 package sh.calaba.instrumentationbackend;
 
-import android.content.Context;
-import android.content.Intent;
-
 import java.io.PrintWriter;
 import java.io.StringWriter;
 
 public class StatusReporter {
-    private Context context;
+
     private boolean hasReportedFailure;
 
-    public StatusReporter(Context context) {
-        this.context = context;
+    public enum FinishedState {SUCCESSFUL, NOT_SUCCESSFUL}
```

Comment:
Nitpick: let's use `SUCCESSFUL` and `FAILED` states 🙃

Reply:
It's our historical legacy :D

Reply:
Generally, I agree with Andrey, but it seems like we use these names as the content of dump files and I'm not 100% sure are we use its content anywhere else so my opinion: let's keep it as is.